changelog/22.0/22.0.0/summary.md

```diff
@@ -155,6 +156,10 @@ While the flag will continue to accept float values (interpreted as seconds) for
 
 - `--consolidator-query-waiter-cap` flag to set the maximum number of clients allowed to wait on the consolidator. The default value is set to 0 for unlimited wait. Users can adjust  this value based on the performance of VTTablet to avoid excessive memory usage and the risk of being OOMKilled, particularly in Kubernetes deployments.
 
+#### <a id="reloading-vttablet-acl"/>VTTablet ACL enforcement and reloading</a>
+
+When a tablet is started with `--enforce-tableacl-config`, the enforcement by exiting is only done on start, not on reload when receiving a SIGHUP. When reloading an invalid ACL, it's logged as error and the active ACL remains unchanged.
```

Comment:
```suggestion
When a tablet is started with `--enforce-tableacl-config` it will exit with an error if the contents of the file are not valid. After the changes made in https://github.com/vitessio/vitess/pull/17485 the tablet will no longer exit when reloading the contents of the file after receiving a SIGHUP. When the file contents are invalid on reload the tablet will now log an error and the active in-memory ACLs remain in effect.
```